gherkin/javascript/test/GherkinLineTest.ts

```diff
@@ -0,0 +1,20 @@
+import * as assert from 'assert'
+import GherkinLine from '../src/GherkinLine'
+
+describe('GherkinLine', function() {
+  function getCellsText(line: string) {
+    const gl = new GherkinLine(line, 1)
+
+    return gl.getTableCells().map(span => span.text)
+  }
+
+  it('getTableCells correctly trims cells content', function() {
```

Comment:
Nitpick: Create a nested `describe("#getTableCells()")` block and use one test per assertion. Instead of saying "correctly" (which is open to interpretation), be explicit, such as `it("trims whitespace after")` etc.

Reply:
Oh nice I did not know we were able to have nested `describe` in mocha, that's great :)